### ISSUE TITLE
dcim/cable search fields missing termination_a_id, termination_b_id

### ISSUE THREAD

**Issue by jifox** [nautobot/nautobot#596]
### Environment
* Python version: 3.8.5
* Nautobot version: 9c57a71265ba (v1.0.3b1)

### Proposed Functionality

Allow to search for dcim/cables in swagger api UI with the input-fields that are required for dcim/cables.

![image](https://user-images.githubusercontent.com/8406301/122767372-e1d06d00-d2a2-11eb-944a-4636dae75f7e.png)

Also GraphQL will complain when trying that fields as filter.

![image](https://user-images.githubusercontent.com/8406301/122768547-fd884300-d2a3-11eb-94cd-8c62e46f691c.png)

### Use Case

The user wants to check if data already exists by providing the required fields of dcim/cables as filter

### Database Changes

None

### External Dependencies

None

**Comment by glennmatthews** [comment]
Thanks for the report!

**Comment by gneville-ot** [comment]
Is there any update on this one?
I'm using pynautobot and wish to check if the cable exists before creating.

This is the code I'm using:

```
source = nb.dcim.interfaces.get(name='et-0/0/1',device='switch1')
target = nb.dcim.interfaces.get(name='et-1/0/1',device='switch2')

nb.dcim.cables.get(termination_a_type="dcim.interface", termination_b_type="dcim.interface", termination_a_id=source.id, termination_b_id=target.id)

```

I've confirmed I get the right IDs for the ports, but the query errors with this:

```
  File "/usr/local/lib/python3.8/dist-packages/pynautobot/core/endpoint.py", line 206, in filter
    raise ValueError("filter must be passed kwargs. Perhaps use all() instead.")
ValueError: filter must be passed kwargs. Perhaps use all() instead.
```

**Comment by bryanculver** [comment]
Closed via #1981
